java/google/registry/export/ExportConstants.java

```diff
@@ -91,7 +42,9 @@
 
   /** Returns the names of kinds to import into reporting tools (e.g. BigQuery). */
   public static ImmutableSet<String> getReportingKinds() {
-    return FluentIterable.from(REPORTING_ENTITY_CLASSES)
+    //Get all classses withe the ReportedOn Annotation @ReportedOn for use in export to BigQuery.
```

Comment:
Nit: typo ("withe") and space is preferred after the //.  I'm not actually sure you even need this comment, the code combined with the existing javadoc is pretty self explanatory I think (the getBackupKinds() logic is slightly more complicated).